include/nccl_ofi_rdma.h

```diff
@@ -296,18 +315,29 @@ typedef struct nccl_ofi_rdma_ep_name {
  * connection information.
  */
 typedef struct nccl_ofi_rdma_connection_info {
-	/* A comm identitifer that uniquely identifies the comm on the sender
-	   side. The receiver must use this ID when sending messages to sender */
-	uint64_t local_comm_id;
+	/* Message type
+	 * either NCCL_OFI_RDMA_MSG_CONN or NCCL_OFI_RDMA_MSG_CONN_RESP
+	 */
+	nccl_ofi_rdma_msg_type_t type;
 
 	/* Number of rails */
 	int num_rails;
 
+	/* A comm identitifer that uniquely identifies the comm on the sender
+	   side. The receiver must use this ID when sending messages to sender */
+	uint32_t local_comm_id;
+
+	/* A comm identitifer that uniquely identifies the comm
+	 * on the receiver side */
+	uint32_t remote_comm_id;
+
 	/* Array of `MAX_NUM_RAILS` `nccl_ofi_rdma_ep_name_t`
 	 * structs. The member `num_rails` indicates the number of
 	 * entries that are in use. */
 	nccl_ofi_rdma_ep_name_t ep_names[MAX_NUM_RAILS];
 } nccl_ofi_rdma_connection_info_t;
+_Static_assert(sizeof(nccl_ofi_rdma_connection_info_t) == 240,
+	       "Wrong size for RDMA connect message");
```

Comment:
`connection_info_t` isn't critical path, do we really need this static assert?

Reply:
Also, I'd put some context as to why we have these static asserts, otherwise the next dev will simply change the number when the assert breaks and not realize the reason for the assert is about padding.

Reply:
even if it is not critical path, I would leave the static assert, to make it clear that these structs go on the wire. I will add a comment to it.